### PR TITLE
Remove automation of resampling

### PR DESCRIPTION
**Pull request by JoelClemence**
* Migrate trigger-resample-lambda to Spring Cloud Function

### CODE REVIEW

File: lambdas/trigger-resample-lambda/src/test/java/uk/gov/ukho/ais/triggerresamplelambda/TriggerResampleLambdaApplicationTest.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class TriggerResampleLambdaApplicationTest {
+
+  @Test
+  public void whenFunctionStandsUpThenIsStartedProperly() {}
```
Comment (galexite) [inline lambdas/trigger-resample-lambda/src/test/java/uk/gov/ukho/ais/triggerresamplelambda/TriggerResampleLambdaApplicationTest.java]: Empty test?

Comment (JoelClemence) [reply lambdas/trigger-resample-lambda/src/test/java/uk/gov/ukho/ais/triggerresamplelambda/TriggerResampleLambdaApplicationTest.java]: This is a test that tests that the spring context is started properly - i.e. the beans are configured correctly

Comment (JoelClemence) [reply lambdas/trigger-resample-lambda/src/test/java/uk/gov/ukho/ais/triggerresamplelambda/TriggerResampleLambdaApplicationTest.java]: The test fails if the application can't be started

Comment (galexite) [reply lambdas/trigger-resample-lambda/src/test/java/uk/gov/ukho/ais/triggerresamplelambda/TriggerResampleLambdaApplicationTest.java]: Maybe this should be described in a comment inside the method body?

**Pull request merged**